recipes/qh3/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "qh3" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qh3-{{ version }}.tar.gz
+  sha256: eb527d8317746209509b9c575527577cdc9b3cfb0f49294fc1cd109b0570362c
+
+build:
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - cmake
+    - libclang  # [linux]
```

Comment:
```suggestion
```
This is something on PyPI only to reduce the size of binary artifacts (mostly?). In all recipes I have seen it, I just patched it out as a requirement.

Reply:
I'll try Clang instead then

Reply:
Or maybe I need to use Clang as the C compiler and not gcc?

Reply:
@xhochy It seems that it's during the rust/cargo build phase it needs libclang, but not sure why it's not detecting it